scipy/stats/tests/test_variation.py

```diff
@@ -0,0 +1,134 @@
+import numpy as np
+from numpy.testing import assert_equal, assert_allclose
+import pytest
+from scipy.stats import variation
+
+
+class TestVariation:
+    """
+    Test class for scipy.stats.variation
+    """
+
+    def test_ddof(self):
+        x = np.arange(9.0)
+        assert_allclose(variation(x, ddof=1), np.sqrt(60/8)/4)
+
+    @pytest.mark.parametrize('sgn', [1, -1])
+    def test_sign(self, sgn):
+        x = np.array([1, 2, 3, 4, 5])
+        v = variation(sgn*x)
+        expected = sgn*np.sqrt(2)/3
+        assert_allclose(v, expected, rtol=1e-10)
+
+    def test_scalar(self):
+        # A scalar is treated like a 1-d sequence with length 1.
+        assert_equal(variation(4.0), 0.0)
+
+    @pytest.mark.parametrize('nan_policy, expected',
+                             [('propagate', np.nan),
+                              ('omit', np.sqrt(20/3)/4)])
+    def test_variation_nan(self, nan_policy, expected):
+        x = np.arange(10.)
+        x[9] = np.nan
+        assert_allclose(variation(x, nan_policy=nan_policy), expected)
+
+    def test_nan_policy_raise(self):
+        x = np.array([1.0, 2.0, np.nan, 3.0])
+        with pytest.raises(ValueError, match='input contains nan'):
+            variation(x, nan_policy='raise')
+
+    def test_bad_nan_polciy(self):
```

Comment:
```suggestion
    def test_bad_nan_polciy(self):
    def test_bad_nan_policy(self):
```

Reply:
Ooops.  I'll fix this in the next update.

Reply:
Done.

Reply:
Done.